drivers/hmis/app/graphql/mutations/create_client.rb

```diff
@@ -0,0 +1,35 @@
+module Mutations
+  class CreateClient < BaseMutation
+    argument :input, Types::HmisSchema::ClientInput, required: true
+
+    field :client, Types::HmisSchema::Client, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def resolve(input:)
+      user = hmis_user
+
+      client = Hmis::Hud::Client.new(
+        data_source_id: user.data_source_id,
+        user_id: user.user_id,
+        personal_id: SecureRandom.uuid.gsub(/-/, ''),
+        date_updated: DateTime.current,
+        date_created: DateTime.current,
+        **input.to_params,
+      )
+
+      errors = client.errors
```

Comment:
I _think_ this has be be called after `client.valid?` otherwise it will be blank.